### PR TITLE
lnx_auditd_debugfs_usage.yml

### PR DESCRIPTION
**Pull request by blueteam0ps**

### CODE REVIEW

**Comment by nasbench** [comment]
Can you please provide information regarding the `auditd` rules you used? We can use it as information in the rule. 

Also, note that in the near future the field `timeframe` will be deprecated in favor of the new [SIGMA Correlation](https://github.com/SigmaHQ/sigma-specification)

**Comment by blueteam0ps** [comment]
> Can you please provide information regarding the `auditd` rules you used? We can use it as information in the rule.
> 
> Also, note that in the near future the field `timeframe` will be deprecated in favor of the new [SIGMA Correlations](https://github.com/SigmaHQ/sigma-specification)

Updated the rule

**Comment by nasbench** [comment]
Sorry for the misunderstanding. I was pointing out that the old SIGMA correlation was going to be deprecated (it's not yet). Since SIGMA correlations are still a WIP we still don't accept rules using it. I will revert the rule to use the `timeframe` and review it again.

File: rules/linux/auditd/lnx_auditd_debugfs_usage.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            - 'fdisk'
+            - 'blkid'
+            - 'parted'
+            - 'hwinfo'
+            - 'inxi'
+    selection_2:
+        type: EXECVE
+        a0: 'debugfs'
+    timeframe: 5m
+    condition: 1 of selection_*
```
Comment (frack113) [inline rules/linux/auditd/lnx_auditd_debugfs_usage.yml:32]:
```suggestion
    condition: selection_2 | near selection_1 # requires both
```
As we want the 2 near each other

**Review by frack113 — APPROVED** [review]

**Review by nasbench — APPROVED** [review]

**Pull request merged**